demo/opae-nlb-demo/Dockerfile

```diff
@@ -1,24 +1,13 @@
-# CLEAR_LINUX_BASE and CLEAR_LINUX_VERSION can be used to make the build
-# reproducible by choosing an image by its hash and installing an OS version
-# with --version=:
-# CLEAR_LINUX_BASE=clearlinux@sha256:b8e5d3b2576eb6d868f8d52e401f678c873264d349e469637f98ee2adf7b33d4
-# CLEAR_LINUX_VERSION="--version=29970"
-#
-# This is used on release branches before tagging a stable version.
-# The main branch defaults to using the latest Clear Linux.
-ARG CLEAR_LINUX_BASE=clearlinux:latest
+# Fedora is a recommended distro for OPAE
+# https://opae.github.io/latest
+ARG FEDORA_RELEASE="32"
+FROM fedora:${FEDORA_RELEASE} as builder
 
-FROM ${CLEAR_LINUX_BASE} as builder
-
-ARG CLEAR_LINUX_VERSION=
-
-RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
-    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc devpkg-tbb git
 # Fetch dependencies and source code
-ARG OPAE_RELEASE=1.5.0-2
+ARG OPAE_RELEASE=2.0.2-1
```

Comment:
I can see some newer tags too. Is this the best one?

Reply:
This is a tested one.